ivy_tests/test_ivy/test_frontends/test_tensorflow/test_random.py

```diff
@@ -90,3 +90,33 @@ def test_tensorflow_normal(
         dtype=dtype[0],
         seed=seed,
     )
+
+
+# random_shuffle
+@handle_frontend_test(
+    fn_tree="tensorflow.random.shuffle",
+    value=helpers.tf_tensor(array=[[1, 2], [3, 4], [5, 6]]).ivy_array,
```

Comment:
Instead of a single test case, you should use a test strategy. The one to use here is 

```
helpers.dtype_and_values
```

in place of ``value`` and ``dtype``, then unpack this under ``test_tensorflow_shuffle``.

Reply:
You should look at the way tests are set up for other tensorflow functions, such as in ``test_general_functions.py``. The other two functions here are sampling functions, which are the exception; most are tensor->tensor functions like shuffle.

Reply:
Thanks so much for this information @Nightcrab.
I'm currently working on this! Thanks!